src/System.Linq/tests/ShortCircuitingTests.cs

```diff
@@ -0,0 +1,222 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Linq.Tests
```

Comment:
Thanks for adding these.

Nit: Other than this one blank line, looks like something may have happened with blank lines elsewhere in the file, i.e. there aren't any :smile:  Would you mind adding some between members, as in the other files?